content/_changelogs/10.10.0.md

```diff
@@ -0,0 +1,46 @@
+## 10.10.0
+
+_Released 10/11/2022_
+
+**Features:**
+
+- [`cy.origin()`](/api/commands/origin)'s URL argument must be an exact origin
+  match to the origin of the AUT. Addresses
+  [#23999](https://github.com/cypress-io/cypress/issues/23999),
+  [#23814](https://github.com/cypress-io/cypress/issues/23814),
+  [#24085](https://github.com/cypress-io/cypress/issues/24085),
+  [#22882](https://github.com/cypress-io/cypress/issues/22882), and
+  [#17166](https://github.com/cypress-io/cypress/issues/17166).
+- More accurately send/set cookies in [`cy.origin()`](/api/commands/origin) when
+  `experimentalSessionAndOrigin` is enabled. Addresses
+  [#23551](https://github.com/cypress-io/cypress/issues/23551),
+  [#22670](https://github.com/cypress-io/cypress/issues/22670), and
+  [#23603](https://github.com/cypress-io/cypress/issues/23603).
+- Cypress will no longer watch files when executing component tests in "run"
+  mode. Addresses [#22959](https://github.com/cypress-io/cypress/issues/22959).
+
+**Bugfixes:**
+
+- Cypress should now be able to connect to Firefox v105. Fixes
+  [#23897](https://github.com/cypress-io/cypress/issues/23897).
+- Cypress component tests should now correctly load assets with Angular. Fixes
```

Comment:
Remove "should" thoughout

"Component tests now correctly load assets with Angular."